### PR TITLE
PB122

### PR DESCRIPTION
**Pull request by dantavori**
## Description
Moved PB122 to new validate

### CODE REVIEW

File: demisto_sdk/commands/validate/validators/PB_validators/PB122_does_playbook_have_unhandled_conditions.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        for task in tasks.values():
+            if task.type == PlaybookTaskType.CONDITION and task.conditions:
+                next_tasks: set[str] = {
+                    k.upper()
+                    for k in (task.nexttasks or {}).keys()
+                    if k and k.upper() != "#DEFAULT#"
+                }
+                condition_labels: set[str] = {
+                    str(condition.get("label")).upper()
+                    for condition in task.conditions
```
Comment (AradCarmi) [inline demisto_sdk/commands/validate/validators/PB_validators/PB122_does_playbook_have_unhandled_conditions.py:63]:
```suggestion
                    for condition in task_conditions
```

File: demisto_sdk/commands/validate/validators/PB_validators/PB122_does_playbook_have_unhandled_conditions.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+        return results
+
+    @staticmethod
+    def unhandled_conditions(playbook: ContentTypes) -> dict[str, set[str]]:
+        unhandled_conditions = {}
+        tasks = playbook.tasks
+        for task in tasks.values():
+            if task.type == PlaybookTaskType.CONDITION and task.conditions:
+                next_tasks: set[str] = {
```
Comment (AradCarmi) [inline demisto_sdk/commands/validate/validators/PB_validators/PB122_does_playbook_have_unhandled_conditions.py:56]: maybe change the name to next_default_tasks

File: demisto_sdk/commands/validate/tests/PB_validators_test.py [hunk 3]
```diff
@@ -199,6 +202,78 @@ def test_is_deprecated_with_invalid_description(content_item, expected_result):
...
+def test_does_playbook_have_unhandled_conditions():
+    """
+    Given: A playbook with condition tasks.
+    When:
+    - Some condition option are unhandled.
+    Then:
+    - Ensure the validation failes on the expected errors only.
+    """
+    playbook = create_playbook_object()
+    playbook.tasks = {
```
Comment (AradCarmi) [inline demisto_sdk/commands/validate/tests/PB_validators_test.py:214]: you can move the task creation into the `create_playbook_object`

**Review by AradCarmi — APPROVED** [review]
Great Job!

File: demisto_sdk/commands/validate/tests/PB_validators_test.py (hunk 3 above)
Comment (dantavori) [reply demisto_sdk/commands/validate/tests/PB_validators_test.py:214]: it's used in many other tests, each one has a different value for `tasks` (including the "untouched" playbook)

**Pull request merged**